### DISCUSSION TITLE
菜单层级配置问题

### DISCUSSION THREAD

**Discussion by kk305cs** [vuepress-theme-hope/vuepress-theme-hope#2247] [Q&A] [answered]
主题文档的配置中写到：由于 markdown.extractHeaders.level 的默认值是 [2, 3] ，因此 headerDepth 的默认最大值是 2；
想实现的效果：在右侧菜单栏中展示4级标题
遇到的问题：在 vuepress.config.ts 中配置 markdown.toc.level 为[2, 3, 4]，主题配置 headerDepth 为 3，但并没有显示 h4 标签名。是不是最大只能设置为 2 呢？

**Comment by Mister-Hope** [comment, marked as answer]
你需要先配置anchor确保anchor提取到了4级标题。

https://v2.vuepress.vuejs.org/reference/config.html#markdown-anchor

**Reply by kk305cs** [reply]
![image](https://user-images.githubusercontent.com/26682400/193241481-7033c36f-7800-4846-ae04-b3cad18eddcd.png)
anchor、headers、toc都配置就可以了，谢谢
